src/utils/display.ts

```diff
@@ -30,10 +30,20 @@ const displayAmount = (amount: string | number): string => {
 const makeFinalContent = (
   items: { heading: string; messages: string[] }[]
 ): Panel => {
-  const formattedItems = items.flatMap((item) => {
+  const formattedItems = items.flatMap((item, i) => {
+    if (i === 0) {
+      return [
+        heading(item.heading),
+        ...item.messages.map((m) => {
+          const safeText = m.replace(NEWLINE_AND_CARRIAGE_RETURN_REGEX, ""); // Strip newlines and carriage returns to avoid injected malicious formatting
+          return text(safeText);
+        }),
+      ];
+    }
+
```

Comment:
I think we can cut this duplication and just have one:
```
const formattedItems = items.flatMap((item, i) => {
      return [
        ...(index > 0 ? [divider()] : []),
        heading(item.heading),
        ...item.messages.map((m) => {
          const safeText = m.replace(NEWLINE_AND_CARRIAGE_RETURN_REGEX, ""); // Strip newlines and carriage returns to avoid injected malicious formatting
          return text(safeText);
        }),
      ];
    }
```